src/public/mapUsers.js

```diff
@@ -0,0 +1,27 @@
+const config = {
+  apiKey: 'AIzaSyBw6HZ7Y4J1dATyC4-_mKmt3u0hLRRqthQ',
+  authDomain: 'staywokesignups.firebaseapp.com',
+  databaseURL: 'https://staywokesignups.firebaseio.com',
+  storageBucket: 'staywokesignups.appspot.com',
+  messagingSenderId: '47559178634',
+}
```

Comment:
only vaguely familiar with firebase. does the client-side config need all these keys? can someone else read/write to our firebase data with these keys?